third_party/silabs/efr32_sdk.gni

```diff
@@ -21,16 +21,15 @@ import("${chip_root}/src/app/icd/icd.gni")
 import("${chip_root}/src/lib/lib.gni")
 import("silabs_board.gni")
 
-if (use_rs9116) {
-  wifi_sapi_root = "${chip_root}/third_party/silabs/wiseconnect-wifi-bt-sdk"
-} else if (use_SiWx917) {
-  wifi_sdk_root = "${chip_root}/third_party/silabs/wifi_sdk"
-}
-
 declare_args() {
   # Location of the efr32 SDK.
   efr32_sdk_root = "${chip_root}/third_party/silabs/gecko_sdk"
   sdk_support_root = "${chip_root}/third_party/silabs/matter_support"
+  wiseconnect_sdk_root =
+      "${chip_root}/third_party/silabs/wiseconnect-wifi-bt-sdk"
+  wifi_sdk_root = "${chip_root}/third_party/silabs/wifi_sdk"
+  examples_plat_dir = "${chip_root}/examples/platform/silabs/efr32"
```

Comment:
This should not be in a declare args. It a set folder that users should not have to modify by passing an argument. Just set it in a local variable

Reply:
Moving it away from `declare_args()`